docs/src/pages/guides/FHIRBulkOperations.md

```diff
@@ -0,0 +1,193 @@
+            "valueUri": "https://localhost:9443/source-fhir-server"
+        },
+        {
+            "name": "input",
+            "part": [
+                {
+                    "name": "type",
+                    "valueString": "Patient"
+                },
+                {
+                    "name": "url",
+                    "valueUrl": "test-import.ndjson"
+                }
+            ]
+        },
+        {
+            "name": "storageDetail",
+            "valueString": "ibm-cos"
+        }
+    ]
+}'
+```
+
+#### Example Response
+The response body is populated only on error, the response status code is 202 and the `content-location` is populated with the polling location, such as the response header:
+
+``` sh
+content-location: https://localhost:9443/fhir-server/api/v4/$bulkdata-status?job=JKyViJ5Y0zcqQ2Uv1aBSMw%3D%3D
+```
+
+## Status Operation: $bulkdata-status
+The `StatusOperation` is a BulkData Management endpoint, which uses the `Content-Location`'s http query parameter value for job to manage the Bulk Data job request.
+
+There are two custom features - *Poll Job* and *Delete Job*.
+
+### **$bulkdata-status: Poll Job**
+The response returned is 202 if the job is queued or not yet complete.
+The response returned is 200 if the job is the job is completed.
+
+#### Example
+- Request
+```sh
+curl 'https://localhost:9443/fhir-server/api/v4/$bulkdata-status?job=FvHrLGPv0oKZNyLzBnY5iA%3D%3D' -k -u "fhiruser:change-password" -v
+```
+- Response for `$import` or `$export` not yet complete
+The response code is 202.
+
+- Response for $export upon completion
+```sh
+{
+"transactionTime": "2020/01/20 16:53:41.160 -0500",
+"request": "/$export?_type=Patient",
+"requiresAccessToken": false,
+"output" : [
+  { "type" : "AllergyIntolerance",
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/AllergyIntolerance_1.ndjson",
+    "count": 20},
+  { "type" : "AllergyIntolerance",
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/AllergyIntolerance_2.ndjson",
+    "count": 8},
+  { "type" : "Observation",
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/Observation_1.ndjson",
+    "count": 234},
+  { "type" : "Observation",
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/Observation_2.ndjson",
+    "count": 81}]
+}
+```
+
+- Response for $import upon completion
+```sh
+{
+    "transactionTime": "2020-04-28T13:13:01.366-04:00",
+    "request": "$import",
+    "requiresAccessToken": false,
+    "output": [
+        {
+            "type": "OperationOutcome",
+            "url": "test-import.ndjson_oo_success.ndjson",
+            "count": 3
```

Comment:
do we always get OperationOutcome for successfully-imported resources?

Reply:
Yes, as I understand they do.  Yet, I haven't seen it successfully written, I think it's a policy on the bucket/user I setup.

Reply:
We agreed to verify this as part of https://github.com/IBM/FHIR/issues/1054